bin/download-syntax

```diff
@@ -81,6 +81,12 @@ class Repo(NamedTuple):
 
 # BEGIN
 REPOS = (
+    Repo(
+        name='ClasherKasten/vscode-kotlin',
```

Comment:
can you open a PR for the fix?

Reply:
Opened a [PR](https://github.com/mathiasfrohlich/vscode-kotlin/pull/54) on the original repository.

Reply:
Hi, I'm here from [vscode-kotlin](https://github.com/mathiasfrohlich/vscode-kotlin) you probably want to pull from [nishtahir/language-kotlin](https://github.com/nishtahir/language-kotlin) instead. That project is better maintained.